dependencies/required.txt

```diff
@@ -13,7 +13,8 @@ scikit-learn >= 0.24.1 ; python_version >= "3.7"
 scikit-learn < 1.0 ; python_version < "3.7"
 websockets >= 10.1 ; python_version >= "3.7"
 websockets <= 10.0 ; python_version < "3.7"
-filelock
+filelock ; python_version >="3.7"
```

Comment:
missing space

Reply:
I decide to fix in other PRs... Really don't want to wait the pipeline again.